### DISCUSSION TITLE
Add transition to allowed layer styles?

### DISCUSSION THREAD

**Discussion by altano** [chakra-ui/panda#1081] [Ideas]
If I can set `background` for a layer style, I should probably be able to set `transition-property: background`. Seems like it would be reasonable to add a few of these transition properties. Thoughts?

**Comment by anubra266** [comment]
@altano These is the mapping in the panda preset.
https://github.com/chakra-ui/css-panda/blob/main/packages/preset-base/src/utilities/transitions.ts#L9

**Comment by altano** [comment]
Hey @anubra266, thanks for replying.

I don't see the connection between that map and the types that `defineLayerStyles()` accepts, [`LayerStyleProperty`](https://github.com/chakra-ui/panda/blob/ac07841695b818069f3c4cf503fd1e105389525a/packages/types/src/composition.ts#L72-L101). Is there some way to pass a transition to `defineLayerStyles` that I'm not understanding?

**Comment by anubra266** [comment]
@altano I might have misunderstood your question. transition should be working in layerstyles. Could you give a reproduction of your issue on the panda playground please

**Reply by altano** [reply]
Sure, here you go: https://stackblitz.com/edit/stackblitz-starters-mfixtf?file=panda.config.ts

If you remove the `transition` line in panda.config.ts the error goes away.

**Reply by altano** [reply]
And I think that's because `transition` is not present in [this type](https://github.com/chakra-ui/panda/blob/24e783b358951e233646a67e1898931769da684a/packages/types/src/composition.ts#L72-L101).

**Comment by anubra266** [comment]
@altano Transition is not a layer style property
![CleanShot 2023-08-06 at 16 01 57@2x](https://github.com/chakra-ui/panda/assets/30869823/6e62ebe0-453c-4eab-9e50-2046d31d651f)
